developer/likenft/reference.md

```diff
@@ -0,0 +1,201 @@
+| type             | `CreativeWork`                                     | The `contentMetadata.@type` field in ISCN                 |
+| stakeholder_id   | `did:like:1w6es6du93xmhms60gwwtczr2p2h4cdm8r7kc0f` | The `stakeholders.entity.@id` field in ISCN               |
+| stakeholder_name | `kin`                                              | The `stakeholders.entity.name` field in ISCN              |
+| after            | `1659283200`                                       | Get NFT class created after this UNIX second timestamp    |
+| before           | `1659801600`                                       | Get NFT class created before this UNIX second timestamp   |
+| include_owner    | `true`                                             | Whether to include owner of ISCN when counting sold_count |
+| ignore_list      | `like17m4vwrnhjmd20uu7tst7nv0kap6ee7js69jfrs`      | The addresses to ignore when counting sold count          |
+| limit            | `20`                                               | The limit of response. Default & max: 100                 |
+
+Result is order by the number of NFTs not owned by creator nor `ignore_list`
+
+### Example Response
+
+`/likechain/likenft/v1/ranking?ignore_list=like17m4vwrnhjmd20uu7tst7nv0kap6ee7js69jfrs&limit=1`
+
+```json
+{
+  "classes": [
+    {
+      "id": "likenft1yhsps5l8tmeuy9y7k0rjpx97cl67cjkjnzkycecw5xrvjjp6c5yqz0ttmc",
+      "name": "Writing NFT - 分散式出版 DePub 的發展路徑，NFT 從移民到原住民",
+      "description": "...",
+      "symbol": "WRITING",
+      "uri": "https://api.like.co/likernft/metadata?iscn_id=iscn%3A%2F%2Flikecoin-chain%2FIKI9PueuJiOsYvhN6z9jPJIm3UGMh17BQ3tEwEzslQo%2F3",
+      "uri_hash": "",
+      "config": {
+        "burnable": false,
+        "max_supply": "0",
+        "blind_box_config": null
+      },
+      "metadata": {
+        "url": "https://ckxpress.com/nft-immigrant-to-native/",
+        "name": "分散式出版 DePub 的發展路徑，NFT 從移民到原住民",
+        "@type": "Article",
+        "image": "ar://_GJD1gNuIqXicYNWce__4B8Qqg5tN8FBRSUCXR8kuJE",
+        "version": 1,
+        "@context": "http://schema.org/",
+        "keywords": "DePub,LikeCoin,nft,Numbers",
+        "usageInfo": "",
+        "description": "...",
+        "nft_meta_collection_id": "likerland_writing_nft",
+        "nft_meta_collection_name": "Writing NFT",
+        "nft_meta_collection_descrption": "Writing NFT by Liker Land"
+      },
+      "parent": {
+        "type": "ISCN",
+        "iscn_id_prefix": "iscn://likecoin-chain/IKI9PueuJiOsYvhN6z9jPJIm3UGMh17BQ3tEwEzslQo",
+        "account": ""
+      },
+      "price": 0,
+      "created_at": "2022-08-01T06:33:21Z",
+      "sold_count": 171
+    }
+  ],
+  "pagination": {
+    "count": 1
+  }
+}
+
+```
+
+*Note: because only api-public knows the price. `price` in the response is always 0*
+
+### Example UI
+
+https://lancatlin.github.io/likenft-ranking/ 
+
+[source code](https://github.com/lancatlin/likenft-ranking)
```

Comment:
Now we merged them into one, please put one `Example UI` in the last section instead of one for each